apps/rush/src/actions/PublishAction.ts

```diff
@@ -153,34 +153,21 @@ export default class PublishAction extends CommandLineAction {
   }
 
   private _publishChanges(allPackages: Map<string, RushConfigurationProject>): void {
+    const changeManager: ChangeManager = new ChangeManager(this._rushConfiguration);
     const changesPath: string = path.join(this._rushConfiguration.commonFolder, RushConstants.changeFilesFolderName);
-    const changeFiles: ChangeFiles = new ChangeFiles(changesPath);
-    const allChanges: IChangeInfoHash = PublishUtilities.findChangeRequests(
-      allPackages,
-      changeFiles,
-      this._addCommitDetails.value,
-      this._prereleaseToken);
-    const orderedChanges: IChangeInfo[] = PublishUtilities.sortChangeRequests(allChanges);
-
-    if (orderedChanges.length > 0) {
+    changeManager.load(changesPath,
+      this._prereleaseToken,
+      this._addCommitDetails.value);
```

Comment:
Would anyone really call load() without these parameters?  If not, why should they be optional?  It increases our test surface and could introduce mistakes if people forget to provide one of the parameters.